docs/API.md

```diff
@@ -435,7 +435,7 @@ Options can be configured as a parameter to the [init()](./interfaces/tracer.htm
 | -              | `DD_TRACE_DISABLED_PLUGINS`    | -              | A comma-separated string of integration names automatically disabled when tracer is initialized. Environment variable only e.g. `DD_TRACE_DISABLED_PLUGINS=express,dns`. |
 | clientToken    | `DD_CLIENT_TOKEN`              | -              | Client token for browser tracing. Can be generated in the UI at `Integrations -> APIs`. |
 | logLevel       | `DD_TRACE_LOG_LEVEL`           | `debug`        | A string for the minimum log level for the tracer to use when debug logging is enabled, e.g. `'error'`, `'debug'`. |
-| -              | `DD_TRACE_STARTUP_LOGS`        | `true`         | Enable tracer startup configuration and diagnostic log. |
+| startupLogsEnabled  | `DD_TRACE_STARTUP_LOGS`        | `true`         | Enable tracer startup configuration and diagnostic log. |
```

Comment:
Since other features don't have the `enabled` keyword on the programmatic config, I'd do the same for this one, especially since it doesn't have `enabled` even in the env var.

Then it doesn't look like there was any explicit reason to leave it out so we'll be able to merge.